nmdc_runtime/api/endpoints/util.py

```diff
@@ -57,9 +58,25 @@
 HOSTNAME_EXTERNAL = BASE_URL_EXTERNAL.split("://", 1)[-1]
 
 
+def check_filter(filter_: str):
+    if not filter_.startswith("{") or not filter_.endswith("}"):
+        raise HTTPException(
+            status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+            detail=f"Given `filter` is not valid JSON object, which must start with '{{' and end with '}}'.",
```

Comment:
>Given `filter` is not valid JSON object, ...

I started reading this as a dependent clause; like, "Given that `filter` is not a valid JSON object, ..."

Proposal:

The given `filter` is not a valid JSON object, ...

Reply:
great. done.